<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaskulistaWidget</class>
 <widget class="QWidget" name="LaskulistaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabBar" name="tabs" native="true"/>
     </item>
     <item>
      <widget class="KpDateEdit" name="alkupvm"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> - </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KpDateEdit" name="loppupvm"/>
     </item>
     <item>
      <widget class="QLineEdit" name="viiteEdit">
       <property name="placeholderText">
        <string>Etsi viitenumerolla</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="view">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="naytaNappi">
       <property name="text">
        <string>N채yt채</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/print.png</normaloff>:/pic/print.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="kopioiNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Kopioi</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/kopioi.png</normaloff>:/pic/kopioi.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lahetaNappi">
       <property name="text">
        <string>L채het채</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/mail.png</normaloff>:/pic/mail.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="hyvitysNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Hyvityslasku</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/poista.png</normaloff>:/pic/poista.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="muistutusNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Maksumuistutus</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/punainenkuori.png</normaloff>:/pic/punainenkuori.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="uusiNappi">
       <property name="text">
        <string>Uusi lasku</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/uusitiedosto.png</normaloff>:/pic/uusitiedosto.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="muokkaaNappi">
       <property name="text">
        <string>Muokkaa</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/muokkaa.png</normaloff>:/pic/muokkaa.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaNappi">
       <property name="text">
        <string>Poista</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/roskis.png</normaloff>:/pic/roskis.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KpDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header location="global">QTabBar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
